Dockerfiles/Dockerfile.ssas

```diff
@@ -11,11 +11,13 @@ RUN go get -u github.com/derekparker/delve/cmd/dlv
 RUN go get github.com/BurntSushi/toml
 RUN go get github.com/howeyc/fsnotify
 RUN go get github.com/mattn/go-colorable
+RUN go get github.com/CMSgov/bcda-ssas-app/ssas/service/main
```

Comment:
Interesting.  How does this work?  We're only pulling from `ssas/service/main`?  Is `go get` smart enough to pull everything needed?

Reply:
Yes, thankfully everything we need is pulled!  The `go get` command also builds the `main.go` in the specified directory, which is useful [in the Makefile](https://github.com/CMSgov/bcda-app/blob/e45bcf88e420002476d487269baa31c187aab603/Makefile#L108).  

Incidentally we might want a future ticket for renaming the `main` directory/package for SSAS to `ssas` so that `go get` builds a binary with a less generic name.